parameter_files/fates_params_default.cdl

```diff
@@ -1769,6 +1791,14 @@ data:
 
  fates_hydro_solver = 1 ;
 
+ fates_landuse_grazing_carbon_use_eff = 0. ;
+
+ fates_landuse_grazing_maxheight =  1 ;
+
+ fates_landuse_grazing_nitrogen_use_eff = 0.25 ;
+
+ fates_landuse_grazing_phosphorus_use_eff = 0.5 ;
```

Comment:
Are these _use_eff values just placeholders, or from literature (Sam's paper)?  Why again is the carbon_use_eff = 0 ?